src/NuGet.Clients/NuGet.CommandLine/Commands/PushCommand.cs

```diff
@@ -17,6 +17,12 @@ public class PushCommand : Command
         [Option(typeof(NuGetCommand), "CommandApiKey")]
         public string ApiKey { get; set; }
 
+        [Option(typeof(NuGetCommand), "PushCommandSymbolsSourceDescription")]
+        public string SymbolsSource { get; set; }
```

Comment:
I just saw something. In the help for nuget.exe push:

    -NoSymbols     If a symbols package exists, it will not be pushed to a symbols server.

So nuget already uses plural.

---
In reply to: [68461283](https://github.com/NuGet/NuGet.Client/pull/712#discussion_r68461283) [](ancestors = 68461283,68445440)